tools/extras/install_morfessor.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright  2017  Atlas Guide (Author : Lucas Jo)
+#
+# Apache 2.0
+#
+
+echo "#### installing morfessor"
+dirname=morfessor
+if [ ! -d ./$dirname ]; then
+  mkdir -p ./$dirname
+  git clone https://github.com/aalto-speech/morfessor.git morfessor ||
+    {
+      if [ $? -ne 0 ]; then
```

Comment:
this is confusing -- why you are testing  the return code if you have the guarantee done by  `||`

Reply:
it should be removed